source/get-started/bare-metal-install-server.rst

```diff
@@ -810,36 +832,11 @@ If you have your own custom mirror of |CL|, you can add its URL.
 
 #. Select :kbd:`Confirm`.
 
-   .. figure:: /_figures/bare-metal-install-server/bare-metal-install-server-30.png
-      :scale: 100%
-      :alt: Swupd Mirror
-
-      Figure 30: Swupd Mirror
-
-Optional: Skip to `Finish installation`_.
-
-Assign Hostname
-===============
-
-#. In Advanced Options, select :guilabel:`Assign Hostname`.
-
-#. In :guilabel:`Hostname`, enter the hostname only (excluding the domain).
-
-   .. note::
-
-      Hostname does not allow empty spaces. Hostname must start with an
-      alphanumeric character but may also contain hyphens. Maximum length of
-      63 characters.
-
    .. figure:: /_figures/bare-metal-install-server/bare-metal-install-server-31.png
       :scale: 100%
-      :alt: Assign Hostname
-
-      Figure 31: Assign Hostname
-
-#. Navigate to :kbd:`Confirm` until highlighted.
+      :alt: Swupd Mirror
 
-#. Select :kbd:`Confirm`.
+      Figure 31: Swupd Mirror
```

Comment:
Figure 31 needs to be recaptured to reflect that we now have a "Allow insecure http" checkbox item.

Reply:
Great catch. Will do.

Reply:
Figure 31 replaced now.  Does the new checkbox require explanation? IMHO, no, but please advise.

Reply:
No, I think it is self-explaining.